### PR TITLE
Update manager lint fixes + added option to skip tests if executing against EAP

### PR DESCRIPTION
**Pull request by spriadka**

### CODE REVIEW

File: packages/testsuite/cypress/support/resource-utils.ts [hunk 1]
```diff
@@ -261,6 +267,8 @@ declare global {
        */
 
       formInput(configurationFormId: string, attributeName: string): Chainable<JQuery<HTMLElement>>;
+
+      isEAP(managementEndpoint: string): Chainable<boolean>;
```
Comment (OndrejKotek) [inline packages/testsuite/cypress/support/resource-utils.ts:271]: A javadoc is missing.

Comment (spriadka) [reply packages/testsuite/cypress/support/resource-utils.ts:278]: added javadoc, Can we resolve this one?

**Comment by OndrejKotek** [comment]
LGTM, squash please.

**Comment by spriadka** [comment]
@OndrejKotek Squashed & rebased 🙂

**Pull request merged**